src/plugins/intel_cpu/src/nodes/executors/acl/acl_deconv.hpp

```diff
@@ -0,0 +1,193 @@
+            DEBUG_LOG("AclDeconvExecutor does not support layouts:",
+                      " src[0]=", srcDescs[0]->serializeFormat(),
+                      " src[1]=", srcDescs[1]->serializeFormat(),
+                      " src[2]=", srcDescs[2]->serializeFormat(),
+                      " dst=", dstDescs[0]->serializeFormat());
+            return false;
+        }
+
+        auto srcDims  = srcDescs[0]->getShape().getDims();
+        auto weiDims  = srcDescs[1]->getShape().getDims();
+        // swap input and output channels dimensions to be align with ACL
+        // weights tensor shape is changed because ACL expects [W, H, I, O] tensor while OV uses [I, O, H, W] tensor
+        std::swap(weiDims[0], weiDims[1]);
+        auto dstDims  = dstDescs[0]->getShape().getDims();
+
+        VectorDims biasDims;
+        arm_compute::TensorInfo biasTensorInfo;
+        if (deconvAttrs.withBiases) {
+            biasDims = srcDescs[2]->getShape().getStaticDims();
+            //bias presicion is I32 but ACL requests bias precision as input ones
+            biasTensorInfo = arm_compute::TensorInfo(shapeCast(biasDims), 1,
+                                        precisionToAclDataType(srcDescs[0]->getPrecision()), getAclDataLayoutByMemoryDesc(srcDescs[2]));
+        }
+
+        arm_compute::TensorInfo srcTensorInfo = arm_compute::TensorInfo(shapeCast(srcDims), 1,
+                                              precisionToAclDataType(srcDescs[0]->getPrecision()), getAclDataLayoutByMemoryDesc(srcDescs[0]));
+        arm_compute::TensorInfo weiTensorInfo = arm_compute::TensorInfo(shapeCast(weiDims), 1,
+                                              precisionToAclDataType(srcDescs[1]->getPrecision()), getAclDataLayoutByMemoryDesc(srcDescs[1]));
+        arm_compute::TensorInfo dstTensorInfo = arm_compute::TensorInfo(shapeCast(dstDims), 1,
+                                              precisionToAclDataType(dstDescs[0]->getPrecision()), getAclDataLayoutByMemoryDesc(dstDescs[0]));
+
+        unsigned int pad_l = (deconvAttrs.paddingL.size() > 1) ? std::abs(deconvAttrs.paddingL.at(1)) : std::abs(deconvAttrs.paddingL.at(0));
+        unsigned int pad_r = (deconvAttrs.paddingR.size() > 1) ? std::abs(deconvAttrs.paddingR.at(1)) : std::abs(deconvAttrs.paddingR.at(0));
+        unsigned int pad_t = std::abs(deconvAttrs.paddingL.at(0));
+        unsigned int pad_b = std::abs(deconvAttrs.paddingR.at(0));
+        unsigned int stride_x = (deconvAttrs.stride.size() > 1) ? deconvAttrs.stride.at(1) : deconvAttrs.stride.at(0);
+        unsigned int stride_y = deconvAttrs.stride.at(0);
+        unsigned int kernel_x = (deconvAttrs.kernel.size() > 1) ? deconvAttrs.kernel.at(1) : deconvAttrs.kernel.at(0);
+        unsigned int kernel_y = deconvAttrs.kernel.at(0);
+
+        // After stride=8 up-sampling in ACL Deconvolution layer slower than reference
+        if (stride_x >= 8 || stride_y >= 8) return false;
```

Comment:
I don't see there dilation parameter is validated.

Reply:
@dmitry-gorokhov dillation not use in deconv op, but upsampling base on strides and some conditions

Reply:
That's why we need to check that dilation value are equal default (equal 1 if I remember correcly) and return false if not. Otherwise ACL deconv will work incorrectly.